Makefile

```diff
@@ -1,20 +0,0 @@
-ifndef RELEASE_NAME
```

Comment:
We're probably going to still need the Makefile when we use this alongside our current release process or when we run this through jenkins.